lib/web/servers.go

```diff
@@ -127,3 +128,53 @@ func (h *Handler) getDesktopHandle(w http.ResponseWriter, r *http.Request, p htt
 	// to see the desktop once in the UI, so just take the first one.
 	return ui.MakeDesktop(windowsDesktops[0]), nil
 }
+
+// desktopIsActive checks if a desktop has an active session and if so, returns a one time token that is needed
+// to start a session to said desktop.
```

Comment:
Some questions as I am not very familiar with desktop flows.

- Are we returning a token here?
- What's the difference between this API and the one above `getDesktopHandle`?

Reply:
1. We are not, I forgot to update this comment from prototyping